CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.7.0-rc.0 / 2017-05-19
+
+* [BUGFIX] Enable builds for Solaris/Illumos
```

Comment:
We usually sort things the other way around: CHANGE, FEATURE, ENHANCEMENT, BUGFIX. See prometheus/prometheus for examples.